packages/run-protocol/src/makeTracer.js

```diff
@@ -17,8 +17,12 @@ const makeTracer = (name, enable = true) => {
       return infoTick;
     }
     default: {
-      const debugTick = (...args) => {
-        console.log(key, (debugCount += 1), ...args);
+      const debugTick = (optLog, ...args) => {
+        if (optLog.log) {
+          optLog.log(key, (debugCount += 1), ...args);
+        } else {
+          console.info(key, (debugCount += 1), optLog, ...args);
+        }
```

Comment:
I don't see this used

Reply:
It's used in all the `trace` statements. the version that takes a log source as the first argument is used in test-vaultFactory.

Reply:
Ah, thanks. I was looking for `log` and it was https://github.com/Agoric/agoric-sdk/pull/5092/files#diff-d7aa780281e2f8cb56ac8c6ac94cfaf827df62ef48478949275d7fe9b204725cR258